### PR TITLE
[doc] fix some docs issue

### PR DESCRIPTION
**Pull request by mychaow**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

developer guide has some problem about install third-party depencies about openssl

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by jackwener — DISMISSED** [review]

File: docs/en/developer/developer-guide/be-vscode-dev.md [hunk 1]
```diff
@@ -48,9 +48,9 @@ sudo apt install gcc-10 g++-10
 sudo apt-get install autoconf automake libtool autopoint
 ```
 
-4. install : libssl-dev
+4. install : openssl libssl-dev
 ```
-sudo apt install -y libssl-dev
+sudo apt install openssl libssl-dev
```
Comment (adonis0147) [inline docs/en/developer/developer-guide/be-vscode-dev.md:53]: It is better to keep `-y` argument to run the command without interaction.

```suggestion
sudo apt install -y openssl libssl-dev
```

File: docs/zh-CN/developer/developer-guide/be-vscode-dev.md [hunk 2]
```diff
@@ -49,10 +49,10 @@ sudo apt install gcc-10 g++-10
 sudo apt-get install autoconf automake libtool autopoint
 ```
 
-4. 安装 openssl-devel
+4. 安装 openssl libssl-dev
 
 ```
-sudo apt install -y openssl-devel
+sudo apt install openssl libssl-dev
```
Comment (adonis0147) [inline docs/zh-CN/developer/developer-guide/be-vscode-dev.md:55]:
```suggestion
sudo apt install -y openssl libssl-dev
```

**Review by adonis0147 — DISMISSED** [review]

**Review by jackwener — DISMISSED** [review]
LGTM
cc @hf200012

**Comment by hf200012** [comment]
@mychaow rebase

**Review by jackwener — APPROVED** [review]
LGTM
cc @hf200012

**Review by jackwener — APPROVED** [review]
LGTM

**Pull request merged**